f/setAISkill/f_setAISkill.sqf

```diff
@@ -9,18 +9,11 @@ sleep 2;
 
 // ====================================================================================
 
-// DECLARE VARIABLES AND FUNCTIONS
-
-// If an array of units was passed, the skill change will apply only to the units in the array
-params [["_units", [], [[]]]];
-
-// ====================================================================================
-
 // RUN THE SCRIPT ONLY SERVER SIDE
 
-//If _units is empty, then it is called from init.sqf and should only run on the server.
+//If the argument is an empty array, then it is called from init.sqf and should only run on the server.
 //Otherwise it should be allowed to run on the server or on the headless client.
-if (count _units == 0) then {
+if (count _this == 0) then {
 	if (!isServer) exitWith {};
 } else {
 	if (!isServer && hasInterface) exitWith {};
```

Comment:
Don't know if this should be done under a separate PR or not (it's a small change), but this needs to be able to be run on clients to support setting the skill level of Zeus placed units
https://github.com/shadow-fa/F3/blob/f8bf58cbe01812f933e40b94c87d6ce3410cf73b/f/zeus/fn_zeusInit.sqf#L77

Alternatively the above could be `remoteExec`ed, which should be a seperate PR

Reply:
Have you tested and confirmed that it's not working?
Zeus calls `f_fnc_setAISkill` which refers to the file `fn_setAISkill.sqf`. This file does not have any isServer checks. Only the other one `f_setAISkill.sqf`, which is used to set the AISkill for multiple units at once.
The idea is that `f_setAISkill.sqf` gets called on the server (or HC) for setting the skill for all (their) AI at once, while `fn_setAISkill.sqf` can still be used to set the skill for individual units (e.g. for zeus).
I know it's confusing, but it's been like this forever.
(It would probably be less confusing if we'd rename confusing files like this and add all scripts to `CfgFunctions`.)

Reply:
No, I haven't checked whether it does work or not. Seems weird to have a script file that assigns multiple units at once and a function that does a single unit instead of just a single function that takes an array of units, but whatever, it is the way it is.

Thanks for volunteering to renamed confusingly named files and adding everything to `CfgFunctions` :stuck_out_tongue: